tools/ci_connector_ops/ci_connector_ops/pipelines/utils.py

```diff
@@ -175,6 +219,14 @@ def get_modified_connectors(modified_files: Set[str]) -> Set[Connector]:
     return set(modified_connectors)
 
 
+def get_modified_metadata_files(modified_files: Set[str]) -> Set[Path]:
+    return {Path(f) for f in modified_files if f.endswith(METADATA_FILE_NAME) and f.startswith("airbyte-integrations/connectors")}
+
+
+def get_all_metadata_files() -> Set[Path]:
```

Comment:
Hmm its stuff like this thats making me wish the pipeline was located in `airbyte-ci` already so we could have this logic in metadata-service/lib and import it

Reply:
Can you add a TODO here for that?

Ive added an issue to discuss in grooming on the subject: [Move pipelines into the airbyte-ci project](https://app.zenhub.com/workspaces/connector-operations-6318ede5eda32e0010ef8566/issues/gh/airbytehq/airbyte/25170)

Reply:
It's more or less what I planned for https://github.com/airbytehq/airbyte/issues/23975

Reply:
I added the todo.
But I'm not sure this is a great idea use it in the metadata service lib. It would mean that the command would have to be run from the airbyte root repo. I like the fact that this command is agnostic from its environment: pass it a file path and upload/validate it. I think it's better to consider that the pipeline is in a context where all the metadata file resides, not the CLI tool itself.

Reply:
> It would mean that the command would have to be run from the airbyte root repo.

Ah good point! I had been thinking maybe it becomes a `get_all_metadata_files_at_path` type of command.

But as I type it out that does feel a bit silly to leave a TODO for.

Sorry for not thinking about that clearly.